PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.hdinsight/src/com/microsoft/azuretools/hdinsight/projects/HDInsightsJavaProjectWizard.java

```diff
@@ -252,6 +262,11 @@ public void configureJavaProject(String newProjectCompliance, IProgressMonitor m
 	            ((BuildPathsBlock) r).configureJavaProject(newProjectCompliance, new SubProgressMonitor((IProgressMonitor)monitor, 5));
 		            
 	            addMoreSourcetoClassPath();
+	            
+	            HDInsightsJavaProjectWizard parent = (HDInsightsJavaProjectWizard)getWizard();
+	            if (parent != null) {
+	            	parent.canFinish = true;
```

Comment:
Can this assignment be put into `HDInsightsJavaProjectWizard` constructor?

Reply:
Updated to use status check instead of using variable

---
In reply to: [140409437](https://github.com/Microsoft/azure-tools-for-java/pull/1041#discussion_r140409437) [](ancestors = 140409437)